core/templates/dev/head/domain/story/StorySummaryObjectFactorySpec.ts

```diff
@@ -16,39 +16,31 @@
  * @fileoverview Tests for StorySummaryObjectFactory.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { TestBed } from '@angular/core/testing';
+import {
+  StorySummaryObjectFactory,
+  StorySummary
+} from './StorySummaryObjectFactory';
```

Comment:
Don't use relative paths...do something like `domain/story/`

Reply:
When I do this, VSCode complains that it cannot find the module. Is there a way to fix this?

Reply:
I don't use vscode so I can't say for sure, but some general tips can be:
1. Take a look at this [thread](https://github.com/angular/angular-cli/issues/8138)

Try these two locally(remove them while pushing the code):
 => 1. [Suggestion1](https://github.com/angular/angular-cli/issues/8138#issuecomment-375681018)
 => 2. [Suggestion 2](https://github.com/angular/angular-cli/issues/8138#issuecomment-437508081)

The most general one:
=> Try googling around with `vscode tsconfig paths`

Reply:
I'll look into this by myself after submitting this PR. Closing for now.